cli/ingress-controller/main.go

```diff
@@ -413,10 +413,24 @@ func main() {
 	cacheStores.Plugin = kongPluginInformer.GetStore()
 	informers = append(informers, kongPluginInformer)
 
-	kongClusterPluginInformer := kongInformerFactory.Configuration().V1().KongClusterPlugins().Informer()
-	kongClusterPluginInformer.AddEventHandler(reh)
-	cacheStores.ClusterPlugin = kongClusterPluginInformer.GetStore()
-	informers = append(informers, kongClusterPluginInformer)
+	// iterating through all our CRDs to find one seems like overkill, but it doesn't seem like there's
+	// a client function to check support for a single resource only. Alternatively we could try to infer this
+	// if cliConfig.WatchNamespace is set, since that precludes seeting any KongClusterPlugins
+	supportedKongResources, _ := kubeClient.Discovery().ServerResourcesForGroupVersion("configuration.konghq.com/v1")
```

Comment:
Please use https://github.com/Kong/kubernetes-ingress-controller/blob/main/pkg/apis/configuration/v1/register.go#L19 rather than hardcoding a string here.